### ISSUE TITLE
How evaluate metrics 

### ISSUE THREAD

**Issue by HAMZA12337** [sovit-123/fasterrcnn-pytorch-training-pipeline#81]
hi guys i hope you are doing well please i need to know how you find these results 👍 

num classes=10
training images = 14 298
valid images= 4057 img 

epochs  10 

IoU metric: bbox
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.064
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.128
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.059
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.067
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.078
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.116
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.098
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.140
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.146
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.218
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.164
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.177
![map](https://github.com/sovit-123/fasterrcnn-pytorch-training-pipeline/assets/66895798/22d68522-bac8-403b-b45f-849df44d10e3)

**Comment by aymuos15** [comment]
These results are quite poor. Higher the mAP, higher the detection accuracy. Similarly for precision to identify the object. Your scores are very low. Might be train for a bit longer and see how it goes.
